### PR TITLE
[docs] raise z-index block

### LINKED ISSUE #10841: version flyout menu is hidden by table of contents rightbar in docs
### What is the problem?

1. go to a doc page with many entries in the table of contents
2. click the version flyout menu in the bottom right corner

only on master, due to the theme change

screen share: https://gofile.io/d/hBMQtz (.mov, <10 s)

I haven't tried fixing this yet; it might be as simple as raising the version menu's z-index, or we might need to move the version menu back to the bottom left corner

### PR DESCRIPTION
**Pull request by richardliaw**
## Why are these changes needed?
Version warning gets blocked out.

Screen Shot 2020-09-16 at 4 19 02 PM

closes #10841

### CODE REVIEW

**Review by simon-mo — APPROVED** [review]

File: doc/source/_static/css/custom.css [hunk 1]
```diff
@@ -52,3 +52,8 @@ img.inline-figure {
   /* Make version warning clickable */
   z-index: 1;
 }
+
+/* Adjustment to Version block */
+.rst-versions {
+  z-index: 1200;
```
Comment (sumanthratna) [inline doc/source/_static/css/custom.css:58]: I think `z-index: 1030;` will fix this

**Review by sumanthratna — APPROVED** [review]

File: doc/source/_static/css/custom.css (hunk 1 above)
Comment (richardliaw) [reply doc/source/_static/css/custom.css:58]: how did you figure that out?

File: doc/source/_static/css/custom.css [hunk 2]
```diff
@@ -52,3 +52,8 @@ img.inline-figure {
...
+}
```
Comment (richardliaw) [inline doc/source/_static/css/custom.css:59]:
```suggestion
}

```

File: doc/source/_static/css/custom.css (hunk 1 above)
Comment (sumanthratna) [reply doc/source/_static/css/custom.css:58]: haha I couldn't find the z-index of the table of contents so I kind of just binary-searched with trial/error to find the lowest possible z-index

Comment (sumanthratna) [reply doc/source/_static/css/custom.css:58]: I will admit that `1200` instead of `1030` is not likely to cause any problems in the future

Comment (richardliaw) [reply doc/source/_static/css/custom.css:58]: haha will keep as is..

**Pull request merged**